### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by DominicAlvarez**
https://gray-ocean-0e1e42d1e.3.azurestaticapps.net/

### CODE REVIEW

**Comment by AntonioGomez12** [comment]
Screen Shot 2023-04-13 at 9 22 59 PM

Great job! Everything is working fine and meets all requirements!

File: assignment1/src/views/GameView.vue [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<script setup lang="ts">
+import HangMan from '../components/HangMan.vue'
+</script>
+
+<template>
+    <main>
+        <HangMan />
+    </main>
```
Comment (mmwoodfo) [inline assignment1/src/views/GameView.vue:8]:
```suggestion
        <HangMan />
```

File: assignment1/src/components/HangMan.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            Sed at tincidunt orci, quis fringilla purus. 
+            Quisque sed cursus ligula, quis faucibus est. 
+            Praesent nec elit nisi. 
+            Integer convallis blandit arcu eu luctus. 
+            Quisque a imperdiet sapien. 
+            Nam ac interdum ante. 
+            Aliquam dapibus neque in justo maximus, eget placerat nisl porttitor. 
+            Integer pretium ut ante maximus consectetur. 
+            ed porttitor finibus risus sit amet hendrerit. 
+            Nulla laoreet auctor ante eget bibendum.</v-card-text>
```
Comment (mmwoodfo) [inline assignment1/src/components/HangMan.vue:20]:
```suggestion
        <v-card-text>
            Lorem ipsum dolor sit amet, 
            consectetur adipiscing elit. Sed a sem 
            luctus arcu tincidunt condimentum. 
            Ut hendrerit nisi ante, 
            sed fermentum purus 
            commodo tristique. Cras pharetra 
            sodales sapien, in ultrices urna semper vel. 
            Sed at tincidunt orci, quis fringilla purus. 
            Quisque sed cursus ligula, quis faucibus est. 
            Praesent nec elit nisi. 
            Integer convallis blandit arcu eu luctus. 
            Quisque a imperdiet sapien. 
            Nam ac interdum ante. 
            Aliquam dapibus neque in justo maximus, eget placerat nisl porttitor. 
            Integer pretium ut ante maximus consectetur. 
            ed porttitor finibus risus sit amet hendrerit. 
            Nulla laoreet auctor ante eget bibendum.
        </v-card-text>
```
Typically, the tags are on their own lines when there are large blocks of text

File: assignment1/src/components/HangMan.vue [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <v-card-actions><v-btn to="/">Home</v-btn></v-card-actions>
```
Comment (mmwoodfo) [inline assignment1/src/components/HangMan.vue:21]:
```suggestion
        <v-card-actions>
            <v-btn to="/"> Home </v-btn>
        </v-card-actions>
```

**Pull request closed without merging**